### PR TITLE
Brayns plugins

### PR DESCRIPTION
**Pull request by ppodhajski**

### CODE REVIEW

File: var/spack/repos/builtin/packages/brayns-graphs/package.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+
+class BraynsGraphs(CMakePackage):
+    """Brayns plugin"""
+    git = "https://github.com/favreau/Brayns-UC-Graphs.git"
+
+    generator = 'Ninja'
+
+    version('develop')
+    version('0.1.0', branch='master')
```
Comment (pramodk) [inline var/spack/repos/builtin/packages/brayns-graphs/package.py]: missing tag? releasing as version 0.1.0 from master branch is not good idea.

File: var/spack/repos/builtin/packages/lunchbox/package.py [hunk 2]
```diff
@@ -18,6 +18,6 @@ class Lunchbox(CMakePackage):
     depends_on('cmake@3.1:', type='build')
     depends_on('ninja', type='build')
 
-    depends_on('boost')
+    depends_on('boost@1.65.0')
```
Comment (pramodk) [inline var/spack/repos/builtin/packages/lunchbox/package.py:21]: is this need to be specific version? or min-max is possible as well? (see the syntax using `:`).

Comment (ppodhajski) [reply var/spack/repos/builtin/packages/lunchbox/package.py:21]: not at all, it was me testing the CI build for this reason:
```
==> Error: An unsatisfiable version constraint has been detected for spec:

    boost@1.68.0%gcc@6.4.0+atomic+chrono~clanglibcpp cxxstd=default +date_time~debug deployment_build=1 +exception+filesystem+graph~icu+iostreams+locale+log+math~mpi+multithreaded~numpy~pic+program_options~python+random+regex+serialization+shared+signals~singlethreaded+system~taggedlayout+test+thread+timer~versionedlayout+wave arch=linux-rhel7-x86_64 


while trying to concretize the partial spec:

    brion
        ^cmake@3.13.0%gcc@6.4.0~doc+ncurses+openssl+ownlibs~qt arch=linux-rhel7-x86_64 
        ^lunchbox
            ^boost@1.68.0%gcc@6.4.0+atomic+chrono~clanglibcpp cxxstd=default +date_time~debug deployment_build=1 +exception+filesystem+graph~icu+iostreams+locale+log+math~mpi+multithreaded~numpy~pic+program_options~python+random+regex+serialization+shared+signals~singlethreaded+system~taggedlayout+test+thread+timer~versionedlayout+wave arch=linux-rhel7-x86_64 
            ^ninja@1.8.2%gcc@6.4.0 arch=linux-rhel7-x86_64 
            ^servus
        ^mvdtool~mpi
            ^hdf5~mpi
                ^zlib@1.2.11%gcc@6.4.0+optimize+pic+shared arch=linux-rhel7-x86_64 
            ^highfive~mpi
        ^vmmlib


brion requires boost version 1.65.0, but spec asked for 1.68.0
Input spec
--------------------------------
 -   brayns
 -       ^embree%intel
 -       ^ospray%intel

```
https://bbpcode.epfl.ch/ci/job/oss.Brayns.github/build_type=Release,platform=volta/2857/console

File: deploy/packages/bbp-packages.yaml [hunk 3]
```diff
@@ -73,6 +73,7 @@ packages:
       - touchdetector@5.3.1
       - brayns@0.8.0 +opendeck +brion ^ospray@1.7.3
       - brayns@immersive +opendeck +brion +optix
+      - brayns-graphs@0.1.0 ^brayns@1.0.0 +brion
```
Comment (matz-e) [inline deploy/packages/bbp-packages.yaml]: Why 1.0.0, and everything else requires 0.8.0?

File: var/spack/repos/builtin/packages/brayns/package.py [hunk 4]
```diff
@@ -14,6 +14,7 @@ class Brayns(CMakePackage):
 
     version('develop', submodules=True)
     version('0.8.0', tag='0.8.0', submodules=True, preferred=True)
+    version('1.0.0', tag='1.0.0', submodules=True)
```
Comment (matz-e) [inline var/spack/repos/builtin/packages/brayns/package.py]: If you are setting 0.8 as preferred, we should be able to remove '^brayns@0.8.0` above from the spec (simpler should be better).

File: var/spack/repos/builtin/packages/brayns-graphs/package.py [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+    git = "https://github.com/favreau/Brayns-UC-Graphs.git"
+
+    generator = 'Ninja'
+
+    version('develop')
+    version('0.1.0', tag='v0.1.0', branch='master')
+
+    depends_on('cmake@3.1:', type='build')
+    depends_on('ninja', type='build')
+    depends_on('brayns')
```
Comment (matz-e) [inline var/spack/repos/builtin/packages/brayns-graphs/package.py]:
```suggestion
    depends_on('brayns@1:')
```
Should then be able to get rid of '^brayns@1.0.0' above

File: deploy/packages/bbp-packages.yaml (hunk 3 above)
Comment (ppodhajski) [reply deploy/packages/bbp-packages.yaml]: not sure if I understand the question. New plugins are working only with latest changes in brayns

**Pull request closed without merging**